core/common/src/main/java/alluxio/resource/DynamicResourcePool.java

```diff
@@ -218,6 +204,7 @@ public DynamicResourcePool(Options options) {
 
     mMaxCapacity = options.getMaxCapacity();
     mMinCapacity = options.getMinCapacity();
+    mResourceAvailable = new ArrayDeque<>(Math.min(mMaxCapacity, 32));
```

Comment:
I see, it is just the default size. It doesn't have to be a constant.

I originally thought it had to match the size with the other `32` size.